backend/serve.go

```diff
@@ -310,7 +347,18 @@ func Manage(pluginID string, serveOpts ServeOpts) error {
 // TestStandaloneServe starts a gRPC server that is not managed by hashicorp.
 // The function returns the gRPC server which should be closed by the consumer.
 func TestStandaloneServe(opts ServeOpts, address string) (*grpc.Server, error) {
-	pluginOpts := GRPCServeOpts(opts)
+	if opts.HandlerMiddlewares == nil {
+		opts.HandlerMiddlewares = make([]HandlerMiddleware, 0)
+	}
+
+	middlewares := defaultHandlerMiddlewares()
+	opts.HandlerMiddlewares = append(middlewares, opts.HandlerMiddlewares...)
+
+	pluginOpts, err := GRPCServeOpts(opts)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
@wbrowne since used in experimental/datasourcetest and by some external plugins, e.g. https://github.com/grafana/google-sheets-datasource/blob/fe8de66f07a9920dbec0ed07836d56850e7ed8bc/pkg/googlesheets/datasource_test.go#L11, adding the default middlewares here. Does this make sense? Tests passes now at least.

Reply:
Yeah that makes sense to me!